### ISSUE TITLE
The XPolaris is not working

### ISSUE THREAD

**Issue by motasem-suleiman** [lhmrosso/XPolaris#2]

**Comment by lschmitt-blip** [comment]
Agreed. I installed R 4.1.0 so that it would be compatible, but I'm getting the error:
Warning message:
package ‘XPolaris’ is not available for this version of R

A version of this package for your version of R might be available elsewhere,
see the ideas at
https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages
